Examples/Executor/Examples.Executor.csproj

```
@@ -40,4 +40,9 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="8.0.1" />
```

Since the second depends on the first, it could just be

```suggestion
    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="8.0.1" />
```

(same for the other examples)